packages/@lwc/synthetic-shadow/src/faux-shadow/html-element.ts

```diff
@@ -5,8 +5,16 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { isDelegatingFocus, isHostElement } from './shadow-root';
-import { hasAttribute, tabIndexGetter, tabIndexSetter } from '../env/element';
-import { isNull, isFalse, defineProperties } from '@lwc/shared';
+import {
+    hasAttribute,
+    innerTextGetter,
+    innerTextSetter,
+    outerTextGetter,
+    outerTextSetter,
+    tabIndexGetter,
+    tabIndexSetter,
+} from '../env/element';
+import { isNull, isFalse, defineProperties, defineProperty } from '@lwc/shared';
```

Comment:
Imports from [external modules](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) should go above local imports. Same for import from `@lwc/features`.